src/cli.rs

```diff
@@ -432,14 +432,14 @@ pub struct Opt {
     )]
     /// Format string for file hyperlinks (requires --hyperlinks).
     ///
-    /// The placeholders "{path}" and "{line}" will be replaced by the absolute file path and the
-    /// line number, respectively. The default value of this option creates hyperlinks using
-    /// standard file URLs; your operating system should open these in the application registered
-    /// for that file type. However, these do not make use of the line number. In order for the link
-    /// to open the file at the correct line number, you could use a custom URL format such as
-    /// "file-line://{path}:{line}" and register an application to handle the custom "file-line" URL
-    /// scheme by opening the file in your editor/IDE at the indicated line number. See
-    /// <https://github.com/dandavison/open-in-editor> for an example.
+    /// The placeholders "{host}", "{path}", and "{line}" will be replaced by the hostname,
+    /// absolute file path and the line number, respectively. The default value of this option
+    /// creates hyperlinks using standard file URLs; your operating system should open these in the
+    /// application registered for that file type. However, these do not make use of the line
+    /// number. In order for the link to open the file at the correct line number, you could use a
+    /// custom URL format such as "file-line://{path}:{line}" and register an application to handle
+    /// the custom "file-line" URL scheme by opening the file in your editor/IDE at the indicated
+    /// line number. See <https://github.com/dandavison/open-in-editor> for an example.
     pub hyperlinks_file_link_format: String,
```

Comment:
I tried to reduce the text a bit, what about:

---

Placeholders "{path}" and "{line}" will be replaced by the absolute file path and the line number; "{host}" with the hostname delta is currently running on.

The default "file:" scheme creates hyperlinks using standard file URI with only the filename, which your terminal or OS should handle. You can specify any scheme, such as "file-line://{path}:{line}" and register an application to handle it. See \<https://dandavison.github.io/delta/hyperlinks.html> for details.

Reply:
Perfect, thanks for doing that, I wasn't happy with it either. I've pushed a version very similar to what you wrote.